example/lib/screen/nav_bar/nav_bar_account_screen.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:xayn_design/xayn_design.dart';
+
+import 'base_nav_bar_screen.dart';
+
+class NavBarAccountScreen extends BaseNavBarScreen {
+  static const routeName = '/navBarAccountScreen';
+
+  const NavBarAccountScreen({Key? key}) : super(key: key);
+
+  @override
+  _NavBarHomeScreenState createState() => _NavBarHomeScreenState();
+}
+
+class _NavBarHomeScreenState extends BaseNavBarScreenState<NavBarAccountScreen>
+    with NavBarConfigsMixin {
+  @override
+  String get screenTitle => 'Account screen';
+
+  @override
+  Color get screenBgColor => Colors.pinkAccent;
+
+  @override
+  List<Widget> buildExtraChildren() => [
+        buildBtn('Open settings', openSettings),
+      ];
+
+  @override
+  NavBarConfigs get navBarConfigs {
+    final search = NavBarItemIconButton(
+      svgIconPath: linden.assets.icons.search,
+      isSelected: false,
+      onPressed: replaceSearch,
+      key: keySearch,
+    );
+    final account = NavBarItemIconButton(
+      svgIconPath: linden.assets.icons.person,
+      isSelected: true,
+      onPressed: () {},
+      key: keyPerson,
+    );
+    final home = NavBarItemIconButton(
+      svgIconPath: linden.assets.icons.home,
+      isSelected: false,
+      onPressed: replaceHome,
+      key: keyHome,
+    );
+
+    return NavBarConfigs([
```

Comment:
💅  the NavbarConfig as singular is more adequate because it is a single config, with various aspects.

Reply:
sure, make sense 😉

Reply:
fixed